scripts/node/config.private.json

```
@@ -0,0 +1,38 @@
+{
+  "mode": "private",
+
+  "readRpcUrl": "https://hoodi.infura.io/v3/<INSERT API KEY>",
+  "privateRelayUrl": "<INSERT PRIVATE URL SHARED BY BLOXROUTE>:8645",
+
+  "privateKey": "<INSERT WALLET PRIVATE KEY>",
+  "expectedChainId": 560048,
+
+  "recipientAddress": "0x0000000000000000000000000000000000000000",
+  "transferAmountEth": "0.25",
+
+  "ethGasLimit": 200000,
+
+  "asset": "WETH",
+  "wethAddress": "0xa8ce0f84459a972d521ba2158f06521ee154d297",
+  "erc20GasLimit": 150000,
+  "wrapGasLimit": 70000,
+  "wrapIfNeeded": true,
+
+  "minPriorityFeeGwei": 10000,
+  "priorityFeeBufferGwei": 0,
+  "retryPriorityFeeBumpGwei": 0,
+  "baseFeeMultiplier": 0,
+
+  "runDurationSecs": 900,
+  "httpTimeoutSecs": 6,
+
+  "complianceFilter": "<INSERT CUSTOM COMPLIANCE LIST NAME>",
+  "validatorListUrl": "<INSERT PRIVATE URL SHARED BY BLOXROUTE>:18550/relay/v1/builder/validators",
```

```suggestion
  "validatorListUrl": "https://bloxroute.hoodi.blxrbdn.com/relay/v1/builder/validators",
```

this doesn't need to be private